### ISSUE TITLE
one example to show lack of robustness

### ISSUE THREAD

**Issue by mayindan** [mayindan/seminar-robustness_AI#1]
briefly introduce the training set, test set
show the current model with accuracy over 99% still classify an image wrong when using complex image.
use this example to motivate that AI systems that work well in lab doesnt work well in real life

TODO:

- [ ] find complicated cases of handwritten digits online
- [ ] run through current model
- [ ] potentially create a UI for the current model to be able to upload photos and then give results (show the process better)
